src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/MemoryLayouts.java

```diff
@@ -144,139 +144,6 @@ private MemoryLayouts() {
      */
     public static final ValueLayout JAVA_DOUBLE = MemoryLayout.ofValueBits(64, ByteOrder.nativeOrder());
 
-    /**
-     * The {@code _Bool} native type.
-     */
-    public static final ValueLayout C_BOOL;
```

Comment:
Can't we keep all these constants as well?

Reply:
I think the idea is to remove all C constants from MemoryLayouts.java (only keep Java layouts etc)